src/grading-settings/assignment-section/index.jsx

```diff
@@ -0,0 +1,223 @@
+            <ol className="course-grading-assignment-items p-0 mb-4">
+              <AssignmentTypeName
+                value={gradeField.type}
+                errorEffort={errorList[`${type}-${idx}`]}
+                onChange={(e) => handleAssignmentChange(e, idx)}
+              />
+              <AssignmentItem
+                className="course-grading-assignment-abbreviation"
+                title={intl.formatMessage(messages.abbreviationTitle)}
+                descriptions={intl.formatMessage(messages.abbreviationDescription)}
+                type="text"
+                name="shortLabel"
+                value={gradeField.shortLabel}
+                onChange={(e) => handleAssignmentChange(e, idx)}
+              />
+              <AssignmentItem
+                className="course-grading-assignment-total-grade"
+                title={intl.formatMessage(messages.weightOfTotalGradeTitle)}
+                descriptions={intl.formatMessage(messages.weightOfTotalGradeDescription)}
+                type="number"
+                min={MIN_NUMBER_VALUE}
+                max={MAX_NUMBER_VALUE}
+                errorMsg={intl.formatMessage(messages.weightOfTotalGradeErrorMessage)}
+                name={weight}
+                value={gradeField.weight}
+                onChange={(e) => handleAssignmentChange(e, idx)}
+                errorEffort={errorList[`${weight}-${idx}`]}
+              />
+              <AssignmentItem
+                className="course-grading-assignment-total-number"
+                title={intl.formatMessage(messages.totalNumberTitle)}
+                descriptions={intl.formatMessage(messages.totalNumberDescription)}
+                type="number"
+                min={1}
+                errorMsg={intl.formatMessage(messages.totalNumberErrorMessage)}
+                name={minCount}
+                value={gradeField.minCount}
+                onChange={(e) => handleAssignmentChange(e, idx)}
+                errorEffort={errorList[`${minCount}-${idx}`]}
+              />
+              <AssignmentItem
+                className="course-grading-assignment-number-droppable"
+                title={intl.formatMessage(messages.numberOfDroppableTitle)}
+                descriptions={intl.formatMessage(messages.numberOfDroppableDescription)}
+                type="number"
+                min={MIN_NUMBER_VALUE}
+                errorMsg={intl.formatMessage(messages.numberOfDroppableErrorMessage)}
+                name={dropCount}
+                gradeField={gradeField}
+                value={gradeField.dropCount}
+                onChange={(e) => handleAssignmentChange(e, idx)}
+                secondErrorMsg={(
+                  <FormattedMessage
+                    id="course-authoring.grading-settings.assignment.number-of-droppable.second.error.message"
+                    defaultMessage="Cannot drop more {type} assignments than are assigned."
```

Comment:
did you forget to add translations?

Reply:
Also for this text
<img width="1219" alt="image" src="https://github.com/raccoongang/frontend-app-course-authoring/assets/93188219/f6b0109e-a74f-491e-953d-b4c929cda5f1">

Reply:
Added